tests/unit/dataqualityfood.t

```diff
@@ -1885,4 +1885,20 @@ ok(
 	'Soluble fiber + Insoluble fiber exceeds total fiber'
 ) or diag explain $product_ref;
 
+# Test case: Mozzarella category with minimum number of ingredients
+my $product_ref = {
+	categories_tags => ['en:mozzarella'],
+	properties => {
+		minimum_number_of_ingredients => 3,
+	},
```

Comment:
no it does not make sense to have it, it will be taken in taxonomy.

Reply:
I will modify the that right away